LionOnTheWeb/lionontheweb-web
# #50 Pre-commit config migration

This resolves #50 

----

* Migrated pre-commit code from the work done in #28 
* Installed Husky, esLint and Prettier dependencies for pre-commit check 
* Added additional commands to package.json for `prepare`, `pre-install`, `lint`, and `format`

## Testing steps
1. Pull down branch
2. Run `yarn` to install the latest dependencies
3. Run `git commit` to make a test commit and see the pre-commit checks work

## Screenshots
![Screenshot 2023-04-07 at 2 26 31 AM](https://user-images.githubusercontent.com/38708680/230554026-86e499dd-b320-45d0-b5ec-3f1c535d5146.png)
